source/documents/delete-multiple.txt

```diff
@@ -14,4 +14,134 @@ Delete Multiple Documents
    :depth: 1
    :class: singlecol
 
-You can perform bulk delete operation on multiple documents in Compass.
\ No newline at end of file
+You can perform bulk delete operations on multiple documents in Compass
+by using the :guilabel:`Delete Documents` modal. Performing delete with
+the :guilabel:`Delete Documents` modal helps you visualize deletes 
+to your data before you apply them.
+
+About this Task
+---------------
+
+Deleting documents is a permanent action and cannot not be undone. 
+Validate documents in the :guilabel:`Preview` of the 
+:guilabel:`Delete Documents` modal before confirming the 
+delete operation.
+
+Before you Begin
+----------------
+
+The :guilabel:`Delete Documents` modal is available starting
+in Compass version ``1.5.X``. For instructions on updating or installing
+the latest Compass version, see :ref:`download-install`. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Apply a query filter
+
+      From the :guilabel:`Documents` tab, input a query into the 
+      :guilabel:`Query bar`. The filter criteria of the query specified 
+      applies to the documents in the :guilabel:`Delete Documents` 
+      modal. If you need to delete all documents in a collection, 
+      leave the :guilabel:`Query bar` blank.
+
+   .. step:: Open the delete documents modal
+
+      On the :guilabel:`Documents` tab, click the :icon-lg:`Trash` 
+      :guilabel:`Delete` button to display the 
+      :guilabel:`Delete Documents` modal. The following 
+      table summarizes the :abbr:`UI (User Interface)` of the modal:
```

Comment:
```suggestion
      On the :guilabel:`Documents` tab, click the :icon-lg:`Trash` 
      :guilabel:`Delete` button to display the :guilabel:`Delete 
      Documents` modal. It has the following elements:
```

Reply:
I actually prefer the term, display the modal, and have used the verbaige in the other page bulk updated task page.